### PR TITLE
fix typos and docs standards and URL.

### PR DESCRIPTION
**Pull request by wpamitkumar**
- fix typos and docs standards and URL.

### CODE REVIEW

File: php/utils.php [hunk 1]
```diff
@@ -184,7 +185,7 @@ function is_path_absolute( $path ) {
 /**
  * Composes positional arguments into a command string.
  *
- * @param array
+ * @param array $args Composes positional arguments.
```
Comment (schlessera) [inline php/utils.php]:
```suggestion
 * @param array $args Positional arguments to compose.
```

File: php/utils.php [hunk 2]
```diff
@@ -194,7 +195,7 @@ function args_to_str( $args ) {
 /**
  * Composes associative arguments into a command string.
  *
- * @param array
+ * @param array $assoc_args Composes associative arguments.
```
Comment (schlessera) [inline php/utils.php]:
```suggestion
 * @param array $assoc_args Associative arguments to compose.
```

File: php/utils.php [hunk 3]
```diff
@@ -559,11 +560,11 @@ function make_progress_bar( $message, $count, $interval = 100 ) {
  * Additionally, this adds 'http://' to the URL if no scheme was found.
  *
  * @param string $url           The URL to parse.
```
Comment (schlessera) [inline php/utils.php]: Alignment is off for this comment block.

Comment (wpamitkumar) [reply php/utils.php]: alignment is the same at is.

Comment (schlessera) [reply php/utils.php]: Image 2020-01-29 at 8 29 12 AM

Comment (wpamitkumar) [reply php/utils.php]: Issue fixed.

**Comment by schlessera** [comment]
Thanks for the PR, @wpamitkumar !

**Pull request merged**